pages/health-care/medical-information-terms-conditions.md

```diff
@@ -1,6 +0,0 @@
----
-title: Terms and Conditions for Medical Information
-layout: page-react.html
-entryname: terms-and-conditions
-includeBreadcrumbs: true
----
```

Comment:
Migrated here, https://github.com/department-of-veterans-affairs/vets-website/pull/10660/files#diff-149fa98eefee56d16a6a7b4725860daaR5.